templates/cube/index.html

```diff
@@ -27,55 +27,135 @@ <h1>You&rsquo;re a professional.</br>Prove it with CUBE.</h1>
 </section>
 
 <section class="p-strip">
-  <div class="u-fixed-width">
-    <h2>What are Microcertifications?</h2>
-    <p class="p-heading--four">Earn your qualification one topic at a time, at your own pace, in any order.</p>
-    <p>The CUBE curriculum is perfectly portoned to accomodate your busy schedule. It consists of 15 separate microcertifications, each covering a specific subject area, that can be taken in any order and at any time.</p>
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h2>What are Microcertifications?</h2>
+      <p class="p-heading--four">Earn your qualification one topic at a time, at your own pace, in any order.</p>
+      <p class="u-sv3">The CUBE curriculum is perfectly portoned to accomodate your busy schedule. It consists of 15 separate microcertifications, each covering a specific subject area, that can be taken in any order and at any time.</p>
```

Comment:
Is portoned a typo for portioned or just a word I don't know? Google didn't return anything

Reply:
Great spot, thanks!